### ISSUE TITLE
PaddleOCR对于对于标点符号（）识别效果差

### ISSUE THREAD

**Issue by Jeff-Zhao1999** [PaddlePaddle/PaddleOCR#6815]
在windows下使用paddle=2.20使用paddleocr=2.4训练字符识别时，针对点阵字符识别训练出推理模型，然而在长串的字符识别结果中，所有英文数字字符基本都能正确无误的识别。仅有符号（）完全没有识别效果，已确认字典中有（）。训练图片10张，训练配置和识别效果如下
Global:
  debug: false
  use_gpu: true
  epoch_num: 200
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output
  save_epoch_step: 3
  eval_batch_step:
  - 0
  - 2000
  cal_metric_during_train: true
  checkpoints: null
  save_inference_dir: null
  use_visualdl: false
  infer_img: null
  character_dict_path: ppocr/utils/en_dict.txt
  character_type: EN
  max_text_length: 25
  infer_mode: false
  use_space_char: true
  distributed: true
  pretrained_model: 
Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.005
  regularizer:
    name: L2
    factor: 1.0e-05
Architecture:
  model_type: rec
  algorithm: CRNN
  Transform: null
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: small
    small_stride:
    - 1
    - 2
    - 2
    - 2
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 48
  Head:
    name: CTCHead
    fc_decay: 1.0e-05
Loss:
  name: CTCLoss
PostProcess:
  name: CTCLabelDecode
Metric:
  name: RecMetric
  main_indicator: acc
Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/
    label_file_list:
    - ./train_data/train_list.txt
    ratio_list:
    - 1.0
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecAug: null
    - CTCLabelEncode: null
    - RecResizeImg:
        image_shape:
        - 3
        - 32
        - 320
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
  loader:
    shuffle: true
    batch_size_per_card: 256
    drop_last: true
    num_workers: 8
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data
    label_file_list:
    - ./train_data/val_list.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - CTCLabelEncode: null
    - RecResizeImg:
        image_shape:
        - 3
        - 32
        - 320
    - KeepKeys:
        keep_keys:
        - image
        - label
        - length
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 64
    num_workers: 4
![捕获1](https://user-images.githubusercontent.com/68420895/177667121-cbe9f814-b3dc-41d7-9aa4-745b67d52f7d.PNG)
